index.css

```diff
@@ -0,0 +1,485 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Nanum Gothic', verdana, sans-serif;
+  box-sizing: border-box;
+  word-wrap: break-word;
+}
+
+body {
+  background-color: rgb(220, 216, 216);
+}
+
+header {
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+ul {
+  margin-top: 50px;
+  list-style: none;
+}
+
+li {
+  font-size: 12px;
+  line-height: 24px;
+  gap: 20px;
+}
+
+ul,
+.program li,
+.speakers li,
+.programs,
+.speakers {
+  display: flex;
+  width: 100%;
+}
+
+header ul {
+  margin-top: 0;
+}
+
+section {
+  width: 100%;
+  padding: 3%;
+}
+
+footer {
+  display: flex;
+  margin: 30px 10%;
+  gap: 60px;
+}
+
+footer img {
+  width: 150px;
+  height: 100px;
+}
+
+.logo img {
+  width: 100px;
+  height: 30px;
+}
+
+.hsection {
+  padding-top: 200px;
+  background: url("./images/main_big.png");
+}
+
+.speakers-section {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h2 {
+  font-size: 24px;
+  color: rgb(255, 255, 255);
+}
+
+.speakers-section h2 {
+  color: #575757;
+  padding-bottom: 15px;
+}
+
+.head img {
+  width: 75%;
+}
+
+.social-media {
+  background-color: rgb(0, 0, 0);
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.social-media img {
+  width: 20px;
+  height: 20px;
+}
+
+.programs {
+  background: url(./images/bg-program.jpg);
+  align-items: center;
+}
+
+.programs,
+.program {
+  flex-direction: column;
+  gap: 16px;
+}
+
+.program li {
+  background-color: rgba(43, 41, 41, 0.5);
+  align-items: center;
+  color: #fff;
+  padding: 3px 7px;
+}
+
+.date {
+  font-size: 18px;
+  font-weight: bold;
+  font-family: Verdana, sans-serif;
+  width: 90%;
+  display: inline-block;
+  text-align: center;
+  margin-bottom: 4px;
+  letter-spacing: 0;
+  color: rgb(87, 87, 87);
+  margin-top: 30px;
+}
+
+.program img {
+  width: 64px;
+  height: 58px;
+}
+
+.speakers {
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+}
+
+.speaker-img img {
+  width: 130px;
+  height: 130px;
+  padding-left: 18px;
+  padding-top: 18px;
+  background: url("./images/computer.png") no-repeat;
+}
+
+.event-description,
+.location {
+  margin: 17px 12px 0;
+  width: 80%;
+  padding: 16px 8px;
+  line-height: 150%;
+  font-size: 15px;
+  text-align: justify;
+  word-break: break-all;
+  color: rgb(51, 51, 51);
+  border: 2px solid #fff;
+}
+
+.event-description {
+  background-color: rgb(229, 229, 229);
+}
+
+.location {
+  text-align: center;
+  border: none;
+  padding: 0;
+  margin: 0;
+  font-weight: bold;
+}
+
+.line {
+  width: 70px;
+  border-bottom: 1px solid rgb(239, 81, 60);
+}
+
+.speaker {
+  color: #222a32;
+  line-height: 24px;
+}
+
+.speaker-name {
+  font-size: 16px;
+}
+
+.speaker-job {
+  color: #ff4f38;
+  font-size: 13px;
+}
+
+.speaker-achievement {
+  font-size: 13px;
+}
+
+.full-program {
+  align-self: flex-end;
+  margin-bottom: 40px;
+}
+
+.full-link {
```

Comment:
- [ ] I kindly suggest that you use the design of the "JJC GLOBAL SUBMIT 2015" BUTTON For your  button design

| Your Design | Template Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-26 11-27-35](https://user-images.githubusercontent.com/52934128/170470878-ebda4944-ab10-4008-91d7-57870c1df7fa.png)  | ![Screenshot from 2022-05-26 11-27-52](https://user-images.githubusercontent.com/52934128/170470843-168a48bb-13ca-45c6-8893-b4759200572b.png)  |